plugins/trace_api_plugin/include/eosio/trace_api_plugin/data_log.hpp

```diff
@@ -10,4 +10,34 @@ namespace eosio { namespace trace_api_plugin {
       block_trace_v0
    >;
 
+   /**
+    * append an entry to the end of the slice
+    *
+    * @param entry : the entry to append
+    * @param slice : the slice to append entry to
+    * @return the offset in the slice where that entry is written
+    */
+   template<typename DataEntry, typename Slice>
+   uint64_t append_data_log(const DataEntry &entry, Slice &slice) {
```

Comment:
This seems like a duplicate of 
```
class store_provider {
  template<typename DataEntry, typename Slice>
  static uint64_t append_store(const DataEntry &entry, Slice &slice);
}
```

Is there a need to have this duplicate and in a plugin-wide header vs something more specific to the Store sub-system?

Reply:
forgot to cleanup, it is gone now